### ISSUE TITLE
Environment Variables

### ISSUE THREAD

**Issue by helloanoop** [usebruno/bruno#8]
Support Environment Variables at the Collection level.

**Comment by Butochnikov** [comment]
@helloanoop how to use variables in the application?

**Comment by helloanoop** [comment]
@Butochnikov You can use the {{var}} syntax

Right now the envs gets parsed everywhere except the json body.

**Comment by helloanoop** [comment]
Yes, It should be possible

When you click on send request, the interpolation should happen on the query strings
https://github.com/usebruno/bruno/blob/main/packages/bruno-app/src/utils/collections/index.js#L375

**Comment by Butochnikov** [comment]
@helloanoop  tested like this:
GET: {{host}}/version - it didn't work

```
Network Error

Please note that if you are using Bruno on the web, then the api you are connecting to must allow CORS. If not, please use the chrome extension or the desktop app
```

**Comment by helloanoop** [comment]
@Butochnikov Are you using the web app or the Desktop App?

On the webapp, there is no way to overcome the CORS restriction enforced by the browser.
So, if your api does not support CORS, then you need to use the desktop app.

Also note that on the desktop app you have the feature of local collections, where the collections are stored directly on the local filesystem in the location you choose. I explain it in the video https://youtu.be/wwXJW7_qyLA?t=159 at 2:40s

**Comment by helloanoop** [comment]
Chrome extension should also fix this. Its pending review in the chrome web store.

**Comment by helloanoop** [comment]
I also see a bug, Unless you save the request (the env vars are not getting parsed), but it should work after parsing.
You can open developer console to see the outgoing request.
![image](https://user-images.githubusercontent.com/9350733/196757857-2417eb22-452e-44e9-99c1-189c9a5c7aac.png)

**Comment by Butochnikov** [comment]
@helloanoop  I am using a desktop application.
yes, it worked after saving. thanks

**Comment by helloanoop** [comment]
@ButochnikovI have fixed this in the v0.2.0 release that just went out.
Now you should be able to use the env vars without saving the request also.
You can download the latest versions at https://www.usebruno.com/downloads

PS:
In case you love dark more, its included in the v0.2.0 release
Currently its based on vscode dark theme, but plan to support custom themes in the near future.
